newIDE/app/public/external/jfxr/jfxr-main.js

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
Let's run Prettier on this file :)

Reply:
strangely enough, prettier makes it like that some times - splitting that one line into three